server/sql/automate-creation.sh

```diff
@@ -4,7 +4,7 @@ if [ "$1" = "-d" ]
 then
 
   mongoimport --db socobo_dev --collection socobouser --drop --file ./data/create-socobouser.json --jsonArray
-  # TODO: Add Demo Data: mongoimport --db socobo_dev --collection recipe --drop --file ./dev/create-recipe.json
+  mongoimport --db socobo_dev --collection recipes --drop --file ./data/create-recipe.json --jsonArray
```

Comment:
Can we name our collections singularly? It's a collection of recipe or it's a collection of socobouser.

Reply:
yes we can, but usually it's a collection of socobousers since there are multiple users in one collection.